github-git/2-identify-issue.md

```diff
@@ -0,0 +1,121 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(identify-issue)=
+# Identify an issue to work on 
+
+Open issues are where developers track bugs, new features, or other improvements needed in the repo. Whether you have a specific fix in mind or are looking for ideas, you should start by exploring existing open issues.
+
+In this lesson you will create or comment on an issue that you'd like to fix in GitHub repository that you don't own. 
+
+:::{admonition} Activity: Create (or find) an issue for a bug that you'd like to fix 
+
+1. Navigate to the [pyOpenSci example repo](https://github.com/pyOpenSci/pyos-demo-package-contribute). 
+2. Explore the documentation and code. You will find many items that need to be fixed, including spelling errors, typos, and more.
+3. Pick a file that you'd like to work on. Open a new issue specifying what you'd like to fix.
+4. Wait for someone on the pyOpenSci team to respond to you with next steps. 
+
+Once you have submitted an issue and someone has responded positively, you can begin working on the changes in your fork. [You will learn how to fork a repo, next.](fork-repo)
+:::
+
+As you scan the issues, look for beginner-friendly labels. Beginner-friendly labels are a clear message that the project welcomes new contributors. Many projects label issues as **good first issue** or **beginner-friendly** to highlight approachable tasks that are approachable for new contributors.
```

Comment:
maybe something along the lines of "picking up existing issues, even though they might not be related to the current problem you are experiencing, are a good way of getting to know and building rapport with the developers. remember you're trying to propose changes to something potentially very dear to them! a small bugfix for a known issue to signalt hat you can come in peace can go a long way"